src/main/resources/templates/profile/myRecipes.html

```diff
@@ -1,10 +1,25 @@
 <!DOCTYPE html>
-<html lang="en" xmlns:th="https://www.thymeleaf.org/">
-<head>
-    <meta charset="UTF-8"/>
-    <title>Title</title>
+<html lang="en" xmlns:th="https://www.thymeleaf.org/" xmlns="http://www.w3.org/1999/html">
+<head th:replace="fragments :: head">
 </head>
 <body>
-<h1>Welcome!</h1>
+    <header>
+        <nav th:replace="fragments :: header"></nav>
+        <div th:replace="fragments :: sidebar"></div>
+    </header>
+    <div class="mainBody">
+        <button type="button" class="btn btn-secondary btn-lg"><a th:href="@{'/recipe/create'}"></a>Add Recipe</button>
```

Comment:
I tried testing this button to see if it would send me to the create recipe page, however it was not working. I played around a bit, it looks like the <a> element needs to be surrounding the <button> element. like this:

 <a><button>Add Recipe</button></a>
 
 (obviously with all the attributes included)

Reply:
Fixed! Thank you!